packages/amplication-client/src/Entity/RelationFkHolderField.tsx

```diff
@@ -33,8 +33,8 @@ const RelationFkHolderField = ({ entity, ...props }: Props) => {
     return (
       entity &&
       data &&
-      !formik.values?.properties.allowMultipleSelection &&
-      !relatedField.properties.allowMultipleSelection
+      !formik.values?.properties?.allowMultipleSelection &&
+      !relatedField.properties?.allowMultipleSelection
```

Comment:
Why do we now need the `?` after `properties`?

Reply:
@abrl91 because they init will null values, and this component is triggered.